web/client/src/library/components/graph/ModelLineage.tsx

```diff
@@ -440,72 +425,27 @@ function GraphControls({ nodes = [] }: { nodes: Node[] }): JSX.Element {
   }
 
   return (
-    <div className="pl-2 flex items-center text-xs text-neutral-400">
+    <div className="px-2 flex items-center text-xs text-neutral-400 @container">
       <div className="contents">
-        {isNotNil(model) && (
-          <span
-            title={model.displayName}
-            className="mr-2 w-full min-w-[10rem] whitespace-nowrap text-ellipsis overflow-hidden"
-          >
-            <b>Model:</b> {truncate(model.displayName, 50, 25)}
-          </span>
-        )}
-        {isNotNil(highlightedNodes) ?? (
-          <span className="mr-2 whitespace-nowrap">
-            <b>Highlighted:</b> {Object.keys(highlightedNodes ?? {}).length}
-          </span>
-        )}
-        {countSelected > 0 && (
-          <span className="mr-2 whitespace-nowrap">
-            <b>Selected:</b> {countSelected}
-          </span>
-        )}
-        {withImpacted && countSelected === 0 && countImpact > 0 && (
-          <span className="mr-2 whitespace-nowrap">
-            <b>Impact:</b> {countImpact}
-          </span>
-        )}
-        {withSecondary && countSelected === 0 && countSecondary > 0 && (
-          <span className="mr-2 whitespace-nowrap">
-            <b>Secondary:</b> {countSecondary}
-          </span>
-        )}
-        <span className="mr-2 whitespace-nowrap">
-          <b>Active:</b> {countActive}
-        </span>
-        {countVisible > 0 && countVisible !== countActive && (
-          <span className="mr-2 whitespace-nowrap">
-            <b>Visible:</b> {countVisible}
-          </span>
-        )}
-        {countHidden > 0 && (
-          <span className="mr-2 whitespace-nowrap">
-            <b>Hidden:</b> {countHidden}
-          </span>
-        )}
-        {countDataSources > 0 && (
-          <span className="mr-2 whitespace-nowrap">
-            <b>Data Sources</b>: {countDataSources}
-          </span>
-        )}
-        {countCTEs > 0 && (
-          <span className="mr-2 whitespace-nowrap">
-            <b>CTEs:</b> {countCTEs}
-          </span>
-        )}
+        <Popover
+          className={clsx('flex @lg:hidden bg-none border-none px-2 py-1')}
```

Comment:
Do we need clsx here?

Reply:
ahah, no, must have added it and then removed the conditional at some point. I'll fix it